packages/pieces/google-sheets/src/lib/actions/find-rows.ts

```diff
@@ -10,11 +10,44 @@ export const findRowsAction = createAction({
         spreadsheet_id: googleSheetsCommon.spreadsheet_id,
         include_team_drives: googleSheetsCommon.include_team_drives,
         sheet_id: googleSheetsCommon.sheet_id,
-        column_name: Property.ShortText({
-            displayName: 'Column Name',
-            description: 'The name of the column to search in, e.g. "A"',
+        column_name: Property.Dropdown<string>({
+            description: 'Column Name',
+            displayName: 'The name of the column to search in',
             required: true,
-        }),
+            refreshers: ['authentication', 'sheet_id', 'spreadsheet_id'],
+            options: async (context) => {
+                const authentication = context.authentication as OAuth2PropertyValue;
+                const spreadsheet_id = context.spreadsheet_id as string;
+                const sheet_id = context.sheet_id as number;
+                const accessToken = authentication['access_token'] ?? '';
+
+                const sheetName = await googleSheetsCommon.findSheetName(accessToken, spreadsheet_id, sheet_id);
+
+                if (!sheetName) {
+                    throw Error("Sheet not found in spreadsheet");
+                }
+
+                const values = await googleSheetsCommon.getValues(spreadsheet_id, accessToken, sheet_id);
+
+                const ret = [];
+                
+                const firstRow = values[0].values;
```

Comment:
What if there is no first row? I think we shouldn't throw an error maybe show letters?